specificatie/domain.yaml

```diff
@@ -264,14 +273,8 @@ components:
       properties:
         aard:
```

Comment:
Kan het voorkomen een KOZ overgaat in/ontstaat uit meerdere KOZ-en met verschillende aard? Als dat niet het geval is, kunnen isOntstaanUit en isOvergegaanIn verwijzen naar één Filiatie en kan de kadastraalOnroerendeZaak property worden gewijzigd naar een array van kadastraalOnroerendeZaak. Dit scheelt in het herhalen van de aard.

Reply:
Volgens het informatiemodel kan er per isOntstaanUit een andere aard van toepassing zijn, maar we hebben al op meer plekken ontdekt dat het informatiemodel en de implementatie uit elkaar lopen........
Ik denk dat dit antwoord bij @kad-pothot vandaan moet komen.

Reply:
@JohanBoer @fsamwel 
Klopt, er is max 1 filiatie per object mogelijk en de aard is daarvan een attribuut, dus dat kan er ook maar 1 zijn.